### PR TITLE
Ria 2466 fee hearing option

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-2466

### Change description ###
RIA-2466 Fee Hearing Option 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/ia-case-api/values.yaml [hunk 1]
```diff
@@ -42,3 +43,4 @@ java:
         - launch-darkly-sdk-key
         - case-api-POSTGRES-PASS
         - AppInsightsInstrumentationKey
+        - launch-darkly-sdk-key
```
Comment (makowalski) [inline charts/ia-case-api/values.yaml:46]: this key is already in master

File: build.gradle [hunk 2]
```diff
@@ -289,6 +289,7 @@ dependencies {
 
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
     compile group: 'uk.gov.service.notify', name: 'notifications-java-client', version: '3.12.0-RELEASE'
+    compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '4.12.1'
```
Comment (makowalski) [inline build.gradle:292]: it is there already: https://github.com/hmcts/ia-case-api/blob/master/build.gradle#L298

File: charts/ia-case-api/values.yaml (hunk 1 above)
Comment (Lee365) [reply charts/ia-case-api/values.yaml]: Removed

File: build.gradle (hunk 2 above)
Comment (Lee365) [reply build.gradle]: Removed

**Pull request closed without merging**